### PR TITLE
(bugfix) Disallow crouch/turnaround from runbrake when sticky walking

### LINKED ISSUE #106: Disallow crouch from run/runbrake when sticky walking
To allow for easier, more consistent sticky walking

### PR DESCRIPTION
**Pull request by jobrien97**
Disallow Squat/Turn transition term during first 20f of RunBrake, when sticky walk flag is on

Ultimately makes sticky walking/boost running easier on GC controller

Fixes #106

### CODE REVIEW

File: fighters/common/src/function_hooks/transition.rs [hunk 1]
```diff
@@ -1,6 +1,14 @@
 use super::*;
 use globals::*;
 
+
+pub fn get_fighter_common_from_accessor<'a>(boma: &'a mut BattleObjectModuleAccessor) -> &'a mut L2CFighterCommon {
```
Comment (techyCoder81) [inline fighters/common/src/function_hooks/transition.rs]: why is this just randomly shoved here?
This should be in utils or something, though im pretty sure it was mentioned that there was a built-in call for this already.

cc @blu-dev

**Review by techyCoder81 — DISMISSED** [review]

File: fighters/common/src/function_hooks/transition.rs (hunk 1 above)
Comment (jobrien97) [reply fighters/common/src/function_hooks/transition.rs]: Was added there temporarily until @blu-dev 's tag PR gets merged (it's included in utils there)

Will make another commit when that gets merged

File: fighters/common/src/function_hooks/change_status.rs [hunk 2]
```diff
@@ -3,19 +3,16 @@ use globals::*;
...
-    if boma.is_fighter() {
-        if [*FIGHTER_STATUS_KIND_RUN, *FIGHTER_STATUS_KIND_RUN_BRAKE].contains(&StatusModule::status_kind(boma)) && !CancelModule::is_enable_cancel(boma) {
-            if [*FIGHTER_STATUS_KIND_WAIT, *FIGHTER_STATUS_KIND_TURN].contains(&next_status) {
-                return 0;
-            }
-        }
-    }
+unsafe fn change_status_request_from_script_hook(boma: &mut BattleObjectModuleAccessor, status_kind: i32, arg3: bool) -> u64 {
+    let mut next_status = status_kind;
+    println!("change status from: {}", StatusModule::status_kind(boma));
```
Comment (blu-dev) [inline fighters/common/src/function_hooks/change_status.rs]: Remove the printlns here please, we don't want these in release

**Review by blu-dev — DISMISSED** [review]
please remove printlns

**Review by black-calculus — APPROVED** [review]
lgtm chief merge away

**Pull request merged**